aml-development/frontend
# Create edit input validation

This adds validation to submission and saving. 

Submission has the same rules as saving, but saving has no required fields meaning that reuse of submission validation cannot be done. 

To test:
create a new listing. attempt to save for every field. 
once complete, submit. 
also test a listing with only submits and no saving. 
test to make sure existing listings load and can be modified (excluding tags)
